cfgov/jinja2/v1/youth_employment_success/goals.html

```diff
@@ -0,0 +1,91 @@
+{% from "goals/goal.html" import renderTextarea with context %}
+{% from 'atoms/radio-button.html' import render with context %}
+{% set labels = ['3 to 6 months', '6 to 9 months', '9 months to 1 year', '1 year or more'] %}
+
+<h1 class="h1">Planning how you’ll get to work</h1>
+<p class="lead-paragraph u-mt0">Let’s make a plan for how you’ll get to work—getting to work on time is important for keeping your job. Having a plan and a backup option can help you be prepared so you don’t get in a bind. This tool will help you figure out your best option for getting to work and how to budget for it.</p>
```

Comment:
Could use `block block__flush-top` in place of `u-mt0`

Reply:
Why use an additional class and the modifier over the atomic class? Just curious about the methodology.

Reply:
The `u-` classes are utility classes for arbitrary adjustments (to the layout, in this case). The `block` classes precede the atomic classes we added, so the utility and block classes aren't really atomic, but the block classes designate a specific thing with a 60 or 30 or 0 pixel margin or padding (depending on the modifiers), whereas the utility classes may apply to anything, and are therefore the more amorphous and less atomic of the two. Should we migrate the layout at some point (e.g. to a CSS grid) it will be easier to find and migrate layout blocks if they are using block classes versus arbitrary margin adjustments. Sometimes the block classes do get ridiculously long, but that's balanced against having a consistent set of layout class structures (which, granted, isn't always consistent).

Reply:
The main issue for me was that by adding the block class I get margins on both the top and the bottom of the element, which means that in addition to the `flush__top` class, I also needed to adjust the bottom margin as well, via another class. Since I don't want zero bottom margin (per the designs) I would have needed to use a `u-mbx` class anyway.

I do see your point here though